# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/import-maps/acquiring/modulepreload-link-header.html.ini
# Commit: 110862c2c4da
# Full Hash: 110862c2c4da9f9cb2b6ecc64668fac0b6275350
# Author: Yoshi Cheng-Hao Huang <allstars.chh@gmail.com>
# Date: 2022-06-28 09:16:40
# Regressor Bug: 1774175
# File Overlap Count: 1
# Description:
#   Bug 1774175 - Call SetAcquiringImportMaps(false) when processing modulepreload. r=yulia,smaug
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D149371
# ==============================================================================

diff -r 2841a1ebc685 -r 110862c2c4da testing/web-platform/meta/import-maps/acquiring/modulepreload-link-header.html.ini
--- a/testing/web-platform/meta/import-maps/acquiring/modulepreload-link-header.html.ini	Mon Jun 27 21:19:51 2022 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,7 +0,0 @@
-[modulepreload-link-header.html]
-  expected: TIMEOUT
-  [With modulepreload link header, import maps should fire error events]
-    expected: NOTRUN
-
-  [With modulepreload link header, import maps are not effective]
-    expected: FAIL